### ISSUE TITLE
[FR] Rapid Security Response

### ISSUE THREAD

**Issue by mrmte** [grahampugh/erase-install#416]
**PLEASE NOTE THAT FEATURE REQUESTS ARE ONLY CONSIDERED AGAINST v.28.0+**

**Is your feature request related to a problem? Please describe.**
Is there a way to add a functionality so it will download and install RSRs? apple just released one and i dont think the erae and install will install that, it will only pull the lastest available mac installers.

**Describe the solution you'd like**
to see if there is a way to install the RSRs

**Describe alternatives you've considered**
mdm forced push installs

**Additional context**

**Comment by grahampugh** [comment]
No, that is not possible. Erase-install only installs full installers. RSRs will never be included in full installers so there's nothing we can do here. You may want to look into SUPERMAN or MDM commands to push RSRs.

By the way, Apple pulled the RSR due to a bug in it causing browsing problems, so you might be glad you didn't push it!
